packages/@webex/internal-plugin-metrics/src/business/business-metrics.ts

```diff
@@ -0,0 +1,31 @@
+import GenericMetrics from '../generic-metrics';
+import {BUISNESS_LOG_IDENTIFIER} from './config';
+import {EventPayload} from '../metrics.types';
+
+/**
+ * @description Util class to handle Buisness Metrics
+ * @export
+ * @class BusinessMetrics
+ */
+export default class BusinessMetrics extends GenericMetrics {
+  /**
+   * Submit a buisness metric to our metrics endpoint.
+   * @param {string} name of the metric
+   * @param {EventPayload} user payload of the metric
+   * @returns {Promise<any>}
+   */
+  public submitBusinessEvent({name, payload}: {name: string; payload: EventPayload}) {
+    const event = {
+      type: ['business'],
```

Comment:
You mean at the GenericMetrics level, then each subclass specify it in the ctor, i.e. `this.type = 'business'` ?

Sure I can do that. I'm just a little worried to add cognitive complexity (i.e. the contract that a subclass of GenericMetrics has to define the metric type in the ctor) because we would still need to use it (i.e. `this.type`) here (as each metrics has a specific shape, so the event is build in each metric implementation anyway).